src/utils/utils.ts

```diff
@@ -325,6 +326,10 @@ export function isExternallyVisible(node: FunctionDefinition): boolean {
   );
 }
 
+export function isReturnParamList(node: ParameterList) {
+  return node.parent instanceof FunctionDefinition && node.parent.vReturnParameters === node;
+}
```

Comment:
It's only used for now in there, but given that Cairo1 introduce that return parameters are not named I think is a condition that will be needed in other places

Reply:
Why would we need it in other places?

Reply:
Because so far made no diference if the param list was from args or from return, perhaps it only affects the writing, but is still a general propose function.

Reply:
Alright, scanned all instances where `vReturnParameters` was used, and right now the only instance where a paremeter list asserts itself as returnParameter is in the writer. Also we never visit a `parameterList` node in any pass (and so it never checks for its parent and so neither checks for its location inside the parent).

So I think it's safe to say the scope of your function is the `ParameterListWriter`